backend/src/api/messages/messages.api.ts

```diff
@@ -0,0 +1,51 @@
+import { Router } from 'express';
+import { validateSchema } from '~/middlewares';
+import { message as validationMessageSchema } from '~/validation-schemas';
+import { ApiPath, HttpCode, MessagesApiPath } from '~/common/enums';
+import { messages as messagesService } from '~/services/services';
+
+const initMessagesApi = (apiRouter: Router): Router => {
+  const chatRouter = Router();
```

Comment:
```suggestion
  const messagesRouter = Router();
```
Rename all `chat` word